### PR TITLE
Update rubygem-smart_proxy_pulp to 1.4.1

### PR DESCRIPTION
**Pull request by jlsherrill**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.21
* [ ] 1.20

See Foreman's [plugin maintainer documentation](https://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

File: packages/plugins/rubygem-smart_proxy_pulp/rubygem-smart_proxy_pulp.spec [hunk 1]
```diff
@@ -79,6 +79,9 @@ cp -pa .%{gem_instdir}/settings.d/pulpnode.yml.example %{buildroot}%{foreman_pro
 
 
 %changelog
+* Fri Mar 08 2019 Justin Sherrill <jsherril@redhat.com>
```
Comment (ekohl) [inline packages/plugins/rubygem-smart_proxy_pulp/rubygem-smart_proxy_pulp.spec]: Looks like a version number is missing here

**Comment by jlsherrill** [comment]
@ekohl fixed!

File: packages/plugins/rubygem-smart_proxy_pulp/rubygem-smart_proxy_pulp.spec [hunk 2]
```diff
@@ -6,23 +6,18 @@
 
 Summary: Basic Pulp support for Foreman Smart-Proxy
 Name: rubygem-%{gem_name}
-Version: 1.3.0
+Version: 1.4.0
 Release: 1%{?dist}
 Group: Applications/System
 License: GPLv3
 URL: https://github.com/theforeman/smart-proxy-pulp
```
Comment (ekohl) [inline packages/plugins/rubygem-smart_proxy_pulp/rubygem-smart_proxy_pulp.spec]: Bonus points if you use the direct URL (https://github.com/theforeman/smart_proxy_pulp)

**Comment by jlsherrill** [comment]
I actually found a bug in this that needs to be fixed in smart-proxy-pulp.  Will update with htat.

**Comment by jlsherrill** [comment]
updated to 1.4.1

**Comment by ekohl** [comment]
Feel free to merge it if you've found no more bugs.

**Comment by jlsherrill** [comment]
Tested in product and looks good :+1:

**Pull request merged**